packages/core/CHANGELOG.MD

```diff
@@ -0,0 +1,22 @@
+# Changelog
```

Comment:
Ditto about having one single changelog for all modules.

Reply:
Ok - I've made that change now - just one ChangeLog for the entire mono-repo.